pyramid_zipkin/zipkin.py

```diff
@@ -199,3 +199,38 @@ def __exit__(self, _exc_type, _exc_value, _exc_traceback):
             binary_annotations=self.binary_annotations,
             span_id=self.span_id,
         )
+
+
+def zipkin_span(
+    service_name,
+    span_name,
+    binary_annotations=None,
+):
+    """Decorator utility for logging a function as a Zipkin span.
+    Note that the decorator doesn't accept timestamp annotations.
+    Any timestamps included in the decorator would likely be way off,
```

Comment:
This was hard to write. The idea is that arguments to function decorators are generally defined at import time. E.g.

```
@zipkin_span('a', 'b', annotations={'something': 10}):
def a():
    do_stuff()
```

"Annotations" in Zipkin are timestamped events, and are necessarily relative to the span they're attached to. I.e. if span A starts at ts 10 and ends at ts 20, any annotations < 10 or > 20 are basically invalid. (I say basically because they'll still show up in the UI, but they'll be confusing, since they're interpreted as "events that happened in the context of this span", but are happening outside the span's context).

Does that make more sense? I'd like to be clearer about this in the docstring if possible.